### DISCUSSION TITLE
posts/music/2024/11/18/01/%ED%85%8C%EC%8A%A4%ED%8A%B8/

### DISCUSSION THREAD

**Discussion by giscus** [melted-mint/fuwari-edited#6] [Comments]
# posts/music/2024/11/18/01/%ED%85%8C%EC%8A%A4%ED%8A%B8/

앞으로 가끔 이런거 올리면 어떨까...

https://meltedmint.com/posts/music/2024/11/18/01/%ED%85%8C%EC%8A%A4%ED%8A%B8/

**Comment by fgghf405** [comment]
화이팅입니다
